### PR TITLE
Cleanup integration tests

### PR DESCRIPTION
**Pull request by Aloren**

### CODE REVIEW

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateDeleteTests.java [hunk 1]
```diff
@@ -83,4 +97,49 @@ public void deleteByObject_returnsFalseIfValueIsAbsent() {
 		Person one = Person.builder().id(id).firstName("tya").emailAddress("gmail.com").build();
 		assertThat(template.delete(one)).isFalse();
 	}
+
+	@Test
+	public void deleteByType_ShouldDeleteAllDocumentsWithCustomSetName() {
+		String id1 = nextId();
+		String id2 = nextId();
+		template.save(new SampleClasses.CustomCollectionClass(id1, "field-value"));
```
Comment (yevtsy) [inline src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateDeleteTests.java]: static imports for classes will be more readable in such cases

**Comment by Aloren** [comment]
@kptfh extracted almost all changes related to integration tests cleanup in this PR.

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+	public void setUp() {
+		super.setUp();
+		deleteAll(Person.class);
+	}
+
+	@Test
+	public void testFindWithFilterEqual() {
+		createIndexIfNotExists(Person.class, "first_name_index", "firstName", IndexType.STRING);
+
+		template.insert(Person.builder().id(nextId()).firstName("Dave").lastName("Matthews").build());
```
Comment (yevtsy) [inline src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java]: why don't just use foreach?

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+		Stream<Person> result = template.find(query, Person.class);
+
+		assertThat(result).hasSize(6);
+	}
+
+	@Test
+	public void testFindWithStatement() {
+		createIndexIfNotExists(Person.class, "first_name_index", "firstName", IndexType.STRING);
+
+		template.insert(new Person(nextId(), "Dave", "Matthews"));
```
Comment (yevtsy) [inline src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java]: the same as above

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+		aerospikeQuery.setFilter(Filter.equal("firstName", "Dave"));
+
+		RecordSet rs = client.query(null, aerospikeQuery);
+		int count = 0;
+		while (rs.next()) {
+			Record r = rs.getRecord();
+			count++;
+		}
+
+		Assert.assertEquals(10, count);
```
Comment (yevtsy) [inline src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java]: you mix this approach with static imports, let's follow one way

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateDeleteTests.java (hunk 1 above)
Comment (Aloren) [reply src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateDeleteTests.java]: fixed

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java (hunk 2 above)
Comment (Aloren) [reply src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java]: reworked the tests completely

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java (hunk 3 above)
Comment (Aloren) [reply src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java]: reworked the tests completely

File: src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java (hunk 4 above)
Comment (Aloren) [reply src/test/java/org/springframework/data/aerospike/core/AerospikeTemplateFindByQueryTests.java]: replaced with assertj assertions

**Pull request merged**